python/pyspark/ml/wrapper.py

```diff
@@ -200,7 +197,6 @@ def _create_model(self, java_model):
     def _fit_java(self, dataset):
         """
         Fits a Java model to the input dataset.
-
```

Comment:
these blank lines should stay.  Even though they are private functions, that's how it's usually done for Sphinx I believe